orc8r/cloud/go/services/orchestrator/obsidian/handlers/gateway_handler_factory.go

```diff
@@ -155,13 +155,14 @@ func GetReadGatewayDeviceHandler(path string, serdes serde.Registry) obsidian.Ha
 				return nerr
 			}
 
+			reqCtx := c.Request().Context()
```

Comment:
We're a little non-uniform with how we're naming/handling the `echo.Context` contexts. Not end of the world, but my preference would be: at the start of every echo handler func that needs to propagate its context, go ahead and grab `ctx := c.Request().Context()` at line 1 of the function, then pass that ctx in everywhere it's required

Reply:
I save the `reqCtx` variable when there's more than 1 place in the callsite that will need to propagate the context, otherwise I just use this inline at the client call

Reply:
In my mind, the context would always be the first defn -- to such a degree that ideally it wouldn't even need to be declared / would have language-level support for implicitly passing ctxs around

But in practice, I don't think it's the end of the world